third_party/boringssl/src/crypto/mem.c

```diff
@@ -135,11 +137,13 @@ static const uint8_t kBoringSSLBinaryTag[18] = {
 };
 
 void *OPENSSL_malloc(size_t size) {
+#if !defined(OPENSSL_SYS_STARBOARD)
```

Comment:
Doesn't that break the `OPENSSL_malloc` and force it to return NULL?

Reply:
I don't think so.
The `if`-guarded block excluded here
isn't present in `mem_starboard.c` at all.